<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>gameSetup</class>
 <widget class="QDialog" name="gameSetup" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>I only play with user who have the following attributes</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3_3" >
     <property name="text" >
      <string>Note: value 0 means ignore its attribute</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="labelBreak" >
       <property name="text" >
        <string>break rate less than or equal to : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbBreak" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="specialValueText" >
        <string>No Limit</string>
       </property>
       <property name="suffix" >
        <string>%</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="labelSpeed" >
       <property name="text" >
        <string>speed less than or equal to : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbSpeed" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="specialValueText" >
        <string>No Limit</string>
       </property>
       <property name="suffix" >
        <string/>
       </property>
       <property name="maximum" >
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="labelUnwelcome" >
       <property name="text" >
        <string>unwelcome less than or equal to : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbUnwelcome" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="specialValueText" >
        <string>No Limit</string>
       </property>
       <property name="suffix" >
        <string/>
       </property>
       <property name="maximum" >
        <number>65535</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3_2" >
       <property name="text" >
        <string>score range:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbRangeMin" >
       <property name="maximum" >
        <number>1000000000</number>
       </property>
       <property name="minimum" >
        <number>-1000000000</number>
       </property>
       <property name="singleStep" >
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelBetween" >
       <property name="text" >
        <string>  --  </string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbRangeMax" >
       <property name="maximum" >
        <number>1000000000</number>
       </property>
       <property name="minimum" >
        <number>-1000000000</number>
       </property>
       <property name="singleStep" >
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbRefuseSameIp" >
     <property name="text" >
      <string>refuse user with same ip</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>gameSetup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>gameSetup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
